### DISCUSSION TITLE
How to customize a podman deployment?

### DISCUSSION THREAD

**Discussion by j-kihet** [uyuni-project/uyuni#9088] [Q&A]
How can i customize the podman deployment of uyuni when using ContainerUtils?
For instance /etc/systemd/system.conf DefaultTasksMax= defaults to 15% witch is only around 300 tasks. 
This is too low for bigger deployments.

**Comment by rjmateus** [comment]
Where do you want to change this? On host OS or in the server container? You should be able to change that file in both places, since it's persistem.

**Comment by j-kihet** [comment]
Sorry if i didn't explain it well, I'm new to containerization.
I would like to make changes to the container running uyuni server.  
Tasks max setting seems to reset after restarts.
